entrypoint/upgradecluster/upgradesuc_test.go

```diff
@@ -33,24 +33,24 @@ var _ = Describe("SUC Upgrade Tests:", func() {
 		)
 	})
 
-	It("Verifies ClusterIP Service pre upgrade", func() {
-		testcase.TestServiceClusterIp(false)
+	It("Verifies ClusterIP Service pre-upgrade", func() {
+		testcase.TestServiceClusterIp(true, false)
 	})
 
 	It("Verifies NodePort Service pre-upgrade", func() {
-		testcase.TestServiceNodePort(false)
+		testcase.TestServiceNodePort(true, false)
 	})
 
 	It("Verifies Ingress pre-upgrade", func() {
-		testcase.TestIngress(false)
+		testcase.TestIngress(true, false)
 	})
 
 	It("Verifies Daemonset pre-upgrade", func() {
-		testcase.TestDaemonset(false)
+		testcase.TestDaemonset(true, false)
 	})
 
 	It("Verifies DNS Access pre-upgrade", func() {
-		testcase.TestDnsAccess(false)
+		testcase.TestDnsAccess(true, false)
 	})
 
```

Comment:
on the upgrade funcs, can we add an step before upgrade like: 

```
	
	It("Prints current state of the cluster", func() {
		shared.PrintClusterState()
	}

```
so we can have a record of that on console.

Reply:
I don't want to do this in a specific `It` clause because that's not a test case and each clause should be specific test cases, but I can include that line as part of the upgrade `It` clause before upgrading, sure

Reply:
sure anywhere

Reply:
Should be resolved